deployment/Dockerfile

```diff
@@ -29,21 +37,21 @@ ENV IP="${IP}"
 ENV PORT="${PORT}"
 ENV DB_PATH="${DB_PATH}"
 
-# hadolint ignore=DL3008
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-    curl \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
 WORKDIR /root/
 
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends ca-certificates \
+    # Clean up
```

Comment:
Are you sure this comment works? IIRC in bash you can't do this with `\` multi-line commands, but it might be okay if you use `&&` for multiple commands.

Reply:
ran the dockerfile locally without any issues